### ISSUE TITLE
Item Descriptions Need Completion

### ISSUE THREAD

**Issue by DarthFutuza** [JKGDevs/JediKnightGalaxies#334]
You can now create item descriptions....which means we need someone to create descriptions for every item in the game.

For example:
![image](https://user-images.githubusercontent.com/4450419/118899052-2dba7b80-b8cb-11eb-9d9c-121c77347c64.png)

To create an item description, locate the item file of the item you want to change.  You can find the item files in this directory: https://github.com/JKGDevs/JediKnightGalaxies/tree/develop/JKGalaxies/ext_data/items

Once you've found it, add the following field:
`"itemDescription": "some description here",`

Keep in mind, the descriptions can only be up to 250 chars in length.  (I suggest using a tool like [this ](https://charactercounttool.com/)to write the descriptions).  This issue is for keeping track of which groups have been done and committed to develop.  

--

- [x] Pistols
- [ ] Carbines
- [x] Repeaters
- [x] Rifles
- [x] Bowcasters
- [x] Snipers
- [x] Mines
- [x] Charges
- [x] Chem (flamethrower etc)
- [x] Launchers (concussion/rockets,etc)
- [x] Arrays (shotguns)
- [x] Ammo
- [x] Grenades
- [x] Armors
- [x] Shields
- [x] Jetpacks
- [x] Consumables

**Comment by DarthFutuza** [comment]
a9c0c9 adds the missing pistol descriptions

**Comment by DarthFutuza** [comment]
So it turns out that while I limited the size of descriptions to 250 characters, there are instances where even that limit can be too long due to the fact that some types of weapons have additional details making them take up too much space and squishing the description area so there is less room.  As a long term fix we should probably make the area scrollable or pageable (eg: arrows allow you to flip through pages of the item description).  I'll make that a separate issue once this is done.
